src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriorsIntegrationTest.java

```diff
@@ -1,139 +1,90 @@
 
-    @Test(enabled = true)
+    @Test
     //use the first 20 variants to save time; they have a nice range of AC from 4 to over 4000
-    public void testUsingDiscoveredAF() {
-        WalkerTestSpec spec = new WalkerTestSpec(
-                "-T CalculateGenotypePosteriors --no_cmdline_in_header" +
-                        " -o %s" +
-                        " -R " + b37KGReference +
+    public void testUsingDiscoveredAF() throws IOException {
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                        " -O %s" +
+                        " -R " + b37_reference_20_21 +    //NOTE: we need a reference for -L
                         " -L 20:10,000,000-10,001,432" +
-                        " -V " + validationDataLocation + "1000G.phase3.broad.withGenotypes.chr20.1MB.vcf",
-                1,
-                Arrays.asList("3e60ca3e04fecf5d6004c08d6f7503ca"));
-        executeTest("testUsingDiscoveredAF", spec);
+                        " -V " + largeDir + "1000G.phase3.broad.withGenotypes.chr20.10100000.vcf",
+                Collections.singletonList(dir + "expectedCGP_testUsingDiscoveredAF.vcf")
+        );
+        spec.executeTest("testUsingDiscoveredAF", this);
     }
 
-    @Test(enabled = true)
+    @Test
     //this test ignores discovered AC and has no external priors, so it should only apply the PP tag with values the same as PLs
     //only test the first 20 variants to save time
-    public void testMissingPriors() {
-        WalkerTestSpec spec = new WalkerTestSpec(
-                "-T CalculateGenotypePosteriors --no_cmdline_in_header -useACoff" +
-                        " -o %s" +
-                        " -R " + b37KGReference +
+    public void testMissingPriors() throws IOException {
+        IntegrationTestSpec spec = new IntegrationTestSpec(
+                        "-useACoff" +
+                        " -O %s" +
+                        " -R " + b37_reference_20_21 +    //NOTE: we need a reference for -L
                         " -L 20:10,000,000-10,001,432" +
-                        " -V " + validationDataLocation + "1000G.phase3.broad.withGenotypes.chr20.1MB.vcf",
-                1,
-                Arrays.asList("1cca249ebc2599c7f24210d4f3204049"));
-        executeTest("testMissingPriors", spec);
+                        " -V " + largeDir + "1000G.phase3.broad.withGenotypes.chr20.10100000.vcf",
+                Collections.singletonList(dir + "expectedCGP_testMissingPriors.vcf")
+        );
+        spec.executeTest("testMissingPriors", this);
     }
 
-    @Test(enabled = true)
-    public void testInputINDELs() {
-        WalkerTestSpec spec = new WalkerTestSpec(
-                "-T CalculateGenotypePosteriors --no_cmdline_in_header -useACoff" +
-                        " -o %s" +
-                        " -R " + b37KGReference +
+    @Test
+    public void testInputINDELs() throws IOException {
```

Comment:
INDELs should be skipped (I can't remember if that means PPs = PLs or PPs are missing)